api/openvpn-rw/read

```diff
@@ -259,6 +259,14 @@ iOS: https://itunes.apple.com/it/app/openvpn-connect/id590379981
     } else {
         error();
     }
+} elsif ($cmd eq 'pending-changes') {
+    # check for pending changes that need a service restart
+    my $pendingChanges = 0;
+
+    if (-e '/var/spool/openvpn/road-warrior-lock') {
+        $pendingChanges = 1;
+    }
+    $ret->{'pendingChanges'} = $pendingChanges;
```

Comment:
We can improve with this:
```perl
$ret->{'pendingChanges'} = (-e '/var/spool/openvpn/road-warrior-lock') ? 1 : 0;
```

@gsanchietti is the perl magician, maybe he can confirm

Reply:
I agree, you can even remove the declaration on line 264: 8 bytes of RAM spared! :D